lib/model/model.go

```diff
@@ -1243,18 +1243,65 @@ func (m *Model) updateLocals(folder string, fs []protocol.FileInfo) {
 		return
 	}
 	files.Update(protocol.LocalDeviceID, fs)
-
+	
 	filenames := make([]string, len(fs))
 	for i, file := range fs {
 		filenames[i] = file.Name
 	}
-
+	
 	events.Default.Log(events.LocalIndexUpdated, map[string]interface{}{
 		"folder":    folder,
 		"items":     len(fs),
 		"filenames": filenames,
 		"version":   files.LocalVersion(protocol.LocalDeviceID),
 	})
+	
+	// Lets us know if file/folder change was originated locally or from a network
+	// sync update.  Now write these to a global log file.
+	if remoteUpdate != true {
```

Comment:
Don't compare to a bool. With the change of the sense of the bool, this would then become

```go
if fromScanning {
    m.localDiskUpdated(...)
}
```